src/SeleniumLibrary/keywords/element.py

```diff
@@ -56,35 +56,54 @@ def element_should_contain(self, locator, expected, message=None):
         The ``message`` argument can be used to override the default error
         message.
 
+        The ``ignore_case`` argument can be set to True to compare case
+        insensitive, default is False.
+
         Use `Element Text Should Be` if you want to match the exact text,
         not a substring.
         """
-        actual = self.find_element(locator).text
+        actual = actual_before = self.find_element(locator).text
+        expected_before = expected
+
```

Comment:
We are not in favor of code blocks. Please remove the empty line from this line and from lines 71 and 76.